src/components/spaces/breeding/breeding-data.tsx

```diff
@@ -0,0 +1,77 @@
+import { inject, observer } from "mobx-react";
+import * as React from "react";
+import { BaseComponent, IBaseProps } from "../../base";
+import { BreedingDataNestPanel } from "./breeding-data-nest-panel";
+// @ts-ignore
+import * as colors from "../../../components/colors.scss";
+import "./breeding-data.sass";
+
+interface IProps extends IBaseProps {}
+interface IState {}
+
+interface LegendItem {
+  label: string;
+  color: string;
+}
+const sexLegend: LegendItem[] = [
+  {label: "Female", color: colors.colorChartYellow},
+  {label: "Male", color: colors.colorChartRed}
+];
+const colorLegend: LegendItem[] = [
+  {label: "Light Brown", color: colors.colorDataMouseBrownLightRep},
+  {label: "Medium Brown", color: colors.colorDataMouseBrownMediumRep},
+  {label: "Dark Brown", color: colors.colorDataMouseBrownDarkRep}
+];
+const genotypeLegend: LegendItem[] = [
+  {label: "R<sup>L</sup>R<sup>L</sup> Mice", color: colors.colorDataMouseBrownLightRep},
+  {label: "R<sup>L</sup>R<sup>D</sup> Mice", color: colors.colorDataMouseBrownMediumRep},
+  {label: "R<sup>D</sup>R<sup>L</sup> Mice", color: colors.colorDataMouseBrownMediumRep},
+  {label: "R<sup>D</sup>R<sup>D</sup> Mice", color: colors.colorDataMouseBrownDarkRep}
+];
+const chartInfo = {
+  color: {legend: colorLegend, title: "Fur Colors" },
+  genotype: {legend: genotypeLegend, title: "Genotypes" },
+  sex: {legend: sexLegend, title: "Sex" }
+};
```

Comment:
I don't have strong feelings about if some of these constant objects (used to create the footer legend) should belong at a higher level (maybe even in the breeding store).